### DISCUSSION TITLE
推荐：An Introduction to Category Theory, Abstraction and Algebra

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#332] [1️⃣ 新鲜事]
一个不错的数学知识的视频：https://www.youtube.com/watch?v=FQYOpD7tv30

特别是对抽象（Abstraction）的介绍，以及深入浅出的教学法，很有启发~

**Comment by tyn1998** [comment]
抽象再抽象，本质再本质：Category Theory （范畴论）

![image](https://github.com/X-lab2017/open-wonderland/assets/32434520/1f23a4ea-a730-4a90-8f8d-672567feb95d)
